### ISSUE TITLE
Issue with angle brackets in markdown documentation

### ISSUE THREAD

**Issue by sodface** [project-repo/cagebreak#68]
https://github.com/project-repo/cagebreak/blob/5dbefccb59d93d3b9683fb243458674a9267da56/README.md?plain=1#L41

When following the links to the documentation, they open in preview mode which displays the prettier formatting but angle brackets aren't escaped so they disappear, for example:

```
*workspace <n>*
	Change to <n>-th workspace
```
is displayed as:
```
workspace Change to -th workspace
```

Maybe change the links to plain view, eg. [configuration](https://github.com/project-repo/cagebreak/blob/master/man/cagebreak-config.5.md?plain=1) or perhaps escaping works, eg:

```
*workspace \<n>*
	Change to \<n>-th workspace
```

Escaping works in comments in a quick test but I don't know if that interferes with scdoc to man page conversion?

**Comment by project-repo** [comment]
Hi sodface

Thank you. We have in fact independently discovered this and will push a
fix soon. (It's already fixed locally but logistics delays this for
at least another day.)

cheers
project-repo

**Comment by project-repo** [comment]
This has been released with 2.3.0.

I am closing this issue.

Cheers
project-repo
